addons/website_sale/static/src/js/website_sale.js

```diff
@@ -346,9 +346,8 @@ export const WebsiteSale = publicWidget.Widget.extend(VariantMixin, {
         const min = parseFloat(input.dataset.min || 0);
         const max = parseFloat(input.dataset.max || Infinity);
         const previousQty = parseFloat(input.value || 0, 10);
-        const quantity = (
-            ev.currentTarget.querySelector('i').classList.contains('oi-minus') ? -1 : 1
-        ) + previousQty;
+        const quantity =
+            (ev.currentTarget.classList.contains("css_quantity_minus") ? -1 : 1) + previousQty;
```

Comment:
Please don't do that in stable. Templates are not updated, and this will prevent customers from changing the quantity. 

There is already a PR open to adress the issue: https://github.com/odoo/odoo/pull/221596

cc @vchu-odoo